From e90e0697552d56760ddc9705ec57bfe70a80afdc Mon Sep 17 00:00:00 2001
From: Huibin Hong <huibin.hong@rock-chips.com>
Date: Wed, 8 Jun 2016 16:20:07 +0800
Subject: [PATCH] arm64: dts: rockchip: add burst mode for rk3366

Rk3366 support single and burst mode, and flushp instruction.
But burst mode improve transfer efficiency.

Please refer to:
Commit 8e770f371cc2
"dmaengine: pl330: add burst mode according to dts config"

Change-Id: I5e3fef4684f324dda015c0afd73535c062952fc1
Signed-off-by: Huibin Hong <huibin.hong@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3366.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3366.dtsi b/arch/arm64/boot/dts/rockchip/rk3366.dtsi
index a14689d565bd..d10c0f5e12c7 100644
--- a/arch/arm64/boot/dts/rockchip/rk3366.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3366.dtsi
@@ -710,6 +710,7 @@
 			#dma-cells = <1>;
 			clocks = <&cru ACLK_DMAC_PERI>;
 			clock-names = "apb_pclk";
+			peripherals-req-type-burst;
 		};
 
 		dmac_bus: dma-controller@ff600000 {
@@ -720,6 +721,7 @@
 			#dma-cells = <1>;
 			clocks = <&cru ACLK_DMAC_BUS>;
 			clock-names = "apb_pclk";
+			peripherals-req-type-burst;
 		};
 	};
 
-- 
2.35.3

